### PR TITLE
feat(github): Add checklist to pr template

### PR DESCRIPTION
**Pull request by brkn**
### Description

- Added a checklist to github pr template

### Notes
- [Link to preview of new template](https://github.com/Hipo/hipo-frontend-project-starter/blob/feat/add-checklist-to-pr-template/.github/PULL_REQUEST_TEMPLATE.md)

### CODE REVIEW

**Review by mucahit — APPROVED** [review]

**Review by burakguneli — APPROVED** [review]

**Review by bilgeyucel — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -12,3 +12,13 @@
...
+
+### Checklist:
+
+* ☐ Cleaned up the console log and browsed to see if any errors or warnings popped up
+* ☐ Checked for forgotten todo comments
+* ☐ Checked for forgotten fixme comments
+* ☐ Checked if there is dangling console.log calls
+* ☐ Checked if inner functions definitions are below the return statement of the parent function.
+* ☐ Checked if it should be translated
+* ☐ Checked if any needed id is missing
```
Comment (edizcelik) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
* ☐ Checked if any needed `testid` data attribute is missing
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -12,3 +12,13 @@
 ### Related Tickets
 
 - [#ticket_number](https://hipo.codebasehq.com/projects/PROJECT_SLUG/tickets/ticket_number)
+
+### Checklist:
+
+* ☐ Cleaned up the console log and browsed to see if any errors or warnings popped up
+* ☐ Checked for forgotten todo comments
```
Comment (edizcelik) [inline .github/PULL_REQUEST_TEMPLATE.md]: I don't think TODO comments needs to be added to the checklist

Comment (mucahit) [reply .github/PULL_REQUEST_TEMPLATE.md]: Makes sense, should we remove the FIXME line also, wdyt?

Comment (edizcelik) [reply .github/PULL_REQUEST_TEMPLATE.md]: I use TODO for technical debt kinda messages but I also started using `FIXME` when I'm developing and there is something that I want to come back to and change before PR is merged. So I think `FIXME` can still be included in the checklist

**Pull request merged**